.eslintrc.json

```diff
@@ -65,6 +63,7 @@
     "no-await-in-loop": "off",
     "no-constant-condition": "error",
     "no-return-await": "off",
+    "prefer-arrow-callback": "off",
```

Comment:
This wanted to change all our forwardRef functions to arrow functions, losing the function name. While I typically prefer arrow function callbacks, I like the communication of naming the function where it provides clarification.